telex/client.py

```diff
@@ -48,6 +48,6 @@ def _post(self, path, payload):
             response = self.session.post('{}{}'.format(self.telex_url, path), data=json.dumps(payload))
             response.raise_for_status()
         except requests.exceptions.RequestException as e:
-            raise RequestException(request=e.request, response=e.response)
+            raise RequestException(response=getattr(e, 'response', None))
```

Comment:
the idea was to provide package specific exception types, so a user of the package doesn't need to know about requests. i think thats a pretty common practice, but we're also the only consumers of this package that i'm aware of.